modules/core/client/components/SuccessMessage.js

```diff
@@ -0,0 +1,42 @@
+// External dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+// Internal dependencies
+import sparklesSvg from '../img/sparkles.svg';
+
+const Container = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 500px;
+  padding: 30px;
+  text-align: center;
+`;
+
+const Sparkles = styled.img`
+  margin-bottom: -50px;
+`;
+
+const Spacer = styled.div`
+  margin: 15px 0;
+`;
+
+export default function SuccessMessage({ children, cta, title }) {
+  return (
+    <Container>
+      <Sparkles src={sparklesSvg} alt="" aria-hidden />
+      <h2 className="font-brand-light">{title}</h2>
+      {children && <Spacer>{children}</Spacer>}
```

Comment:
I see that `children` is not used now. Are we going to need it for future uses of `SuccessMessage`?

Reply:
It's used here:

https://github.com/Trustroots/trustroots/blob/ef21faf9c43aeb34399e49b6db815e417fe0c136/modules/references/client/components/create-reference/SubmittedInfo.js#L25-L52

It's all the content between `<SuccessMessage></SuccessMessage>

Reply:
Oh, my bad! Need to work on my react. Thanks for the explanation!